### PR TITLE
[MDCT-423] Section D: Program Characteristics

### PR DESCRIPTION
**Pull request by juliuscaesar**
## Description
Adds Section D: Plan-Level Program Characteristics & Enrollment. Closes [MDCT-423](https://qmacbis.atlassian.net/browse/MDCT-423)

### How to test
http://localhost:3000/mcpar/plan-level-indicators/program-characteristics

### Changed Dependencies
N/A

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

**Review by gmrabian — APPROVED** [review]

File: services/ui-src/src/forms/mcpar/dpc/dpc.json [hunk 1]
```diff
@@ -32,6 +26,26 @@
           "hint": "What is the total number of individuals enrolled in each plan as of the first day of the last month of the reporting year?",
           "mask": "comma-separated"
         }
+      },
+      {
+        "id": "dpc-2",
+        "type": "number",
+        "props": {
+          "label": "D1.I.2 Plan share of Medicaid",
+          "hint": "Automatically calculated. Sum of plan enrollment (within the specific program) as a percentage of the state's total Medicaid enrollment.",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpc/dpc.json:35]: Needs "numerator" and "denominator" bullets, like this. Applies to both `dpc-2` and `dpc-3`.

```suggestion
          "hint": "<p>Automatically calculated. Sum of plan enrollment (within the specific program) as a percentage of the state's total Medicaid enrollment.</p><ul><li>first bullet point</li><li>second bullet point</li></ul>",
```

File: services/ui-src/src/forms/mcpar/dpc/dpc.json [hunk 2]
```diff
@@ -32,6 +26,26 @@
...
+          "mask": "comma-separated",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpc/dpc.json:36]: Remove mask for now. Will make percentage mask later. Applies to both `dpc-2` and `dpc-3`.
```suggestion
```

**Review by braxex — CHANGES_REQUESTED** [review]
Minor changes requested.

You'll need to add styles for the `ul` to `Form.tsx` like this:
Screen Shot 2022-08-23 at 12 58 12

File: services/ui-src/src/forms/mcpar/dpc/dpc.json [hunk 3]
```diff
@@ -32,6 +26,24 @@
           "hint": "What is the total number of individuals enrolled in each plan as of the first day of the last month of the reporting year?",
           "mask": "comma-separated"
         }
+      },
+      {
+        "id": "dpc-2",
+        "type": "number",
+        "props": {
+          "label": "D1.I.2 Plan share of Medicaid",
+          "hint": "<p>Automatically calculated. Sum of plan enrollment (within the specific program) as a percentage of the state's total Medicaid enrollment.</p><ul><li>Numerator: Plan enrollment (D1.I.1)</li><li>Denominator: Statewide Medicaid enrollment (B.I.1)</li><ul><li>System will need to calculate after entry in D1.I.1 (non editable)</li></ul></ul>",
```
Comment (braxex) [inline services/ui-src/src/forms/mcpar/dpc/dpc.json:35]: The nested `<ul>` is not necessary -- just the numerator and denominator. Same comment for `hint` below.

**Review by braxex — CHANGES_REQUESTED** [review]

**Review by braxex — APPROVED** [review]

**Pull request merged**